backend/inspirehep/records/api/mixins.py

```diff
@@ -203,17 +232,106 @@ def update_refs_in_citation_table(self, save_every=100):
                         citer_id=self.model.id,
                         cited_id=reference,
                         citation_date=citation_date,
+                        citation_type=RecordCitationType.citation,
```

Comment:
so you treat this as a default, I think it makes sense to make it explicit at the DB level as suggested above.